Fossil segment doesn't count properly files included in a normal "merge" operation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The `fossil` segment reports incorrectly the number of files added after a `merge` operation. Updates by merge and deletions appear to be reported properly.

In the reported case, the Fossil segment only shows `statcollector ~6 -2`. It should show `statcollector +27 ~6 -2` given the reported changes.

### Theme

Customized theme:

```yaml
# yaml-language-server: $schema=https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json

version: 2
final_space: true
console_title_template: "{{if .Root}}Admin - {{end}}{{ .Shell }} in {{ .Folder }}"
blocks:
- type: prompt
  alignment: left
  segments:
  - type: session
    style: diamond
    background_templates:
    - "{{if .Root}}green{{end}}"
    leading_diamond: 
    trailing_diamond: 
    template: "{{ if .SSHSession }}\ueba9 {{ end }}{{ .UserName }} "
    background: magenta
    foreground: p:white
  - type: os
    style: diamond
    powerline_symbol: 
    trailing_diamond: 
    template: " {{ if .WSL }}WSL at {{ end }}{{.Icon}} "
    background: blue
  - type: path
    style: diamond
    powerline_symbol: 
    trailing_diamond: 
    template: " \uea83 {{ path .Path .Location }} "
    properties:
      style: full
    background: p:orange
    foreground: p:black
  - type: fossil
    style: powerline
    powerline_symbol: 
    foreground_templates:
    - "{{ if .Status.Changed }}cyan{{ end }}"
    background_templates:
    - "{{ if .Status.Changed }}p:yellow{{ end }}"
    - "{{ if .Status.HasConflicts }}p:red{{ end }}"
    properties:
      # branch_max_length: 25.0
      fetch_status: true
      fetch_upstream_icon: true
    background: cyan
    foreground: p:black
  - type: git
    style: powerline
    powerline_symbol: 
    foreground_templates:
    - "{{ if or (.Working.Changed) (.Staging.Changed) }}p:black{{ end }}"
    - "{{ if and (gt .Ahead 0) (gt .Behind 0) }}p:white{{ end }}"
    - "{{ if gt .Ahead 0 }}p:white{{ end }}"
    background_templates:
    - "{{ if or (.Working.Changed) (.Staging.Changed) }}p:yellow{{ end }}"
    - "{{ if and (gt .Ahead 0) (gt .Behind 0) }}p:red{{ end }}"
    - "{{ if gt .Ahead 0 }}#49416D{{ end }}"
    - "{{ if gt .Behind 0 }}#7A306C{{ end }}"
    template: " {{ if .UpstreamURL }}{{ url .UpstreamIcon .UpstreamURL }} {{ end }}{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }} "
    properties:
      branch_max_length: 25.0
      fetch_status: true
      fetch_upstream_icon: true
    background: lightGreen
    foreground: p:black
  - type: root
    style: powerline
    powerline_symbol: 
    template: " \uf0e7 "
    background: green
    foreground: p:white
  - type: shell
    style: powerline
    powerline_symbol: 
    template: " <b>{{ .Name }}</b> "
    background: p:white
    foreground: p:black
    properties:
      mapped_shell_names:
        tcc: TCmd
  - type: time
    style: diamond
    trailing_diamond: 
    template: " {{ .CurrentDate | date \"02/01/2006 15:04:05\" }}"
    background: "#9597DD"
    foreground: p:black
- type: prompt
  alignment: right
  segments:
  - type: node
    style: plain
    template: "\ue718 "
    properties:
      display_mode: files
      fetch_package_manager: false
      home_enabled: false
    background: transparent
    foreground: p:green
  - type: docker
    style: powerline
    powerline_symbol: 
    foreground: "#000000"
    background: "#0B59E7"
    template: "  {{ .Context }} "
  - type: go
    style: plain
    template: "\ue626 "
    properties:
      fetch_version: false
    background: transparent
    foreground: p:blue
  - type: python
    style: plain
    template: "\ue235 {{ .Full }} {{.Venv}} "
    properties:
      display_mode: environment
      fetch_version: true
      fetch_virtual_env: true
      home_enabled: true
    background: transparent
    foreground: p:yellow
- type: prompt
  alignment: left
  segments:
  - type: text
    style: plain
    template: ❯
    foreground: white
  newline: true
tooltips:
- type: aws
  tips:
  - aws
  style: diamond
  leading_diamond: 
  trailing_diamond: 
  template: " \ue7ad {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} "
  properties:
    display_default: true
  background: p:orange
  foreground: p:white
- type: az
  tips:
  - az
  style: diamond
  leading_diamond: 
  trailing_diamond: 
  template: " \uebd8 {{ .Name }} "
  properties:
    display_default: true
  background: p:blue
  foreground: p:white
transient_prompt:
  template: "<p:yellow,transparent>\ue0b6</><,p:yellow> {{ .Folder }} </><p:yellow,transparent>\ue0b0</> "
  background: transparent
  foreground: p:black
secondary_prompt:
  template: "<p:yellow,transparent>\ue0b6</><,p:yellow> > </><p:yellow,transparent>\ue0b0</> "
  background: transparent
  foreground: p:black
palette:
  black: "#262B44"
  blue: "#4B95E9"
  green: "#59C9A5"
  orange: "#FF5501"
  red: "#D81E5B"
  violet: "#5B3C9A"
  white: "#E0DEF4"
  yellow: "#F3AE35"
```

### What OS are you seeing the problem on?

Windows, Linux

### Which shell are you using?

bash, cmd, nu, powershell

### Log output

```Shell
Version: 23.15.1

Shell: pwsh (7.4.5)

Prompt:

εé╢username εé░ εÿ¬ εé░ ε¬â ]8;;file:C:/Users/username/Source/comunidadpy_curiosidades\~\Source\comunidadpy_curiosidades]8;;\ εé░εé░ ε£Ñ statcollector ~6 -2 εé░ pwsh εé░ 30/09/2024 10:05:45εé┤
Γ¥» 

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   0 ms
Os(true)                                  -   0 ms
Path(true)                                -   1 ms
Fossil(true)                              -  63 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms
Node(false)                               -   0 ms
Docker(false)                             -   0 ms
Go(false)                                 -   0 ms
Python(false)                             -   2 ms
Text(true)                                -   0 ms

Run duration: 77.4446ms

Cache path: C:\Users\username\AppData\Local\oh-my-posh

Config path: C:\Users\username\Scripts\TCC\mghsm.omp.yaml

Logs:

[38;2;191;207;240m[DEBUG] [38;2;156;231;201m10:05:45.646 [0mdebug.go:49[38;2;156;231;201m ΓåÆ[0m debug mode enabled
[DEBUG] 10:05:45.649 debug.go:49 ΓåÆ plain mode enabled
[DEBUG] 10:05:45.649 terminal.go:Getenv:164 ΓåÆ C:\Users\username\AppData\Local
[TRACE] 10:05:45.649 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 10:05:45.649 terminal.go:CachePath() - 0s
[DEBUG] 10:05:45.649 terminal.go:70 ΓåÆ loading cache file: C:\Users\username\AppData\Local\oh-my-posh\omp.cache
[DEBUG] 10:05:45.649 terminal.go:70 ΓåÆ loading cache key: upgrade_check
[TRACE] 10:05:45.649 terminal.go(C:\Users\username\AppData\Local\oh-my-posh\omp.cache) - 0s
[DEBUG] 10:05:45.649 terminal.go:Getenv:164 ΓåÆ C:\Users\username\AppData\Local
[TRACE] 10:05:45.649 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 10:05:45.649 terminal.go:CachePath() - 551.4┬╡s
[DEBUG] 10:05:45.649 terminal.go:70 ΓåÆ loading cache file: C:\Users\username\AppData\Local\oh-my-posh\omp.cache.86881fa4-c1a2-4910-897f-848ed963b5cc
[DEBUG] 10:05:45.649 terminal.go:70 ΓåÆ loading cache key: prompt_count_cache
[DEBUG] 10:05:45.649 terminal.go:70 ΓåÆ loading cache key: template_cache
[TRACE] 10:05:45.649 terminal.go(C:\Users\username\AppData\Local\oh-my-posh\omp.cache.86881fa4-c1a2-4910-897f-848ed963b5cc) - 0s
[TRACE] 10:05:45.649 terminal.go:setPromptCount() - 0s
[DEBUG] 10:05:45.649 terminal.go:Getenv:164 ΓåÆ C:\Users\username\Scripts\TCC\mghsm.omp.yaml
[TRACE] 10:05:45.649 terminal.go:Getenv(POSH_THEME) - 0s
[DEBUG] 10:05:45.649 terminal.go:ResolveConfigPath:96 ΓåÆ config set using POSH_THEME: C:\Users\username\Scripts\TCC\mghsm.omp.yaml
[TRACE] 10:05:45.649 terminal.go:ResolveConfigPath() - 0s
[TRACE] 10:05:45.649 terminal.go:Init() - 3.2817ms
[TRACE] 10:05:45.649 terminal.go:Flags() - 0s
[TRACE] 10:05:45.654 load.go:loadConfig() - 4.9649ms
[TRACE] 10:05:45.654 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.654 debug.go:57 ΓåÆ terminal program: Windows Terminal
[DEBUG] 10:05:45.654 debug.go:57 ΓåÆ terminal shell: shell
[DEBUG] 10:05:45.654 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.654 terminal.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 10:05:45.654 terminal_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.654 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:WindowsRegistryKeyValue:215 ΓåÆ ColorizationColor(DWORD): 0xC4847545
[TRACE] 10:05:45.655 terminal.go:Shell() - 0s
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ 7.4.5
[TRACE] 10:05:45.655 terminal.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 10:05:45.655 debug.go:PrintDebug:23 ΓåÆ segment: Title
[DEBUG] 10:05:45.655 text.go:Render:80 ΓåÆ rendering template: {{if .Root}}Admin - {{end}}{{ .Shell }} in {{ .Folder }}
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal_windows.go:Root() - 0s
[TRACE] 10:05:45.655 terminal.go:Shell() - 0s
[TRACE] 10:05:45.655 terminal.go:StatusCodes() - 0s
[TRACE] 10:05:45.655 terminal_windows.go:IsWsl() - 0s
[DEBUG] 10:05:45.655 terminal.go:TemplateCache:651 ΓåÆ environment: [ALLUSERSPROFILE=C:\ProgramData ... trimmed for privacy ...]
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:PathSeparator() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 terminal.go:Pwd:189 ΓåÆ C:\Users\username\Source\comunidadpy_curiosidades
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal_windows.go:IsWsl() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 terminal.go:User:330 ΓåÆ username
[TRACE] 10:05:45.655 terminal.go:User() - 513┬╡s
[DEBUG] 10:05:45.655 terminal.go:Host:347 ΓåÆ ENDAUTOuWKJyWSg
[TRACE] 10:05:45.655 terminal.go:Host() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.655 terminal.go:Getenv(SHLVL) - 0s
[TRACE] 10:05:45.655 terminal.go:TemplateCache() - 513┬╡s
[TRACE] 10:05:45.655 terminal.go:Shell() - 0s
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ 4
[TRACE] 10:05:45.655 terminal.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ 1
[TRACE] 10:05:45.655 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Session
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.655 terminal.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.655 terminal.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 10:05:45.655 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Path
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Time
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Git
[DEBUG] 10:05:45.655 time.go:Enabled:29 ΓåÆ 15:04:05
[DEBUG] 10:05:45.655 path.go:149 ΓåÆ display_cygpath: false
[TRACE] 10:05:45.655 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal.go:PathSeparator() - 0s
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ C:\Users\username\AppData\Local
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[TRACE] 10:05:45.655 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 10:05:45.655 terminal.go:Shell() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Root
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal_windows.go:Root() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Shell
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Fossil
[DEBUG] 10:05:45.655 shell.go:Enabled:28 ΓåÆ mapped_shell_names: map[tcc:TCmd]
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:Shell() - 0s
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.655 terminal.go:Getenv:164 ΓåÆ C:\Users\username\AppData\Local
[TRACE] 10:05:45.655 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.655 terminal.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 10:05:45.655 path.go:setMappedLocations:563 ΓåÆ mapped_locations_enabled: true
[DEBUG] 10:05:45.655 path.go:setMappedLocations:564 ΓåÆ ∩Çô
[DEBUG] 10:05:45.655 path.go:setMappedLocations:565 ΓåÆ ∩Çô
[TRACE] 10:05:45.655 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:PathSeparator() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.655 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 segment.go:SetEnabled:115 ΓåÆ segment: Os
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 os.go:Enabled:34 ΓåÆ εÿ¬
[TRACE] 10:05:45.655 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 path.go:setMappedLocations:566 ΓåÆ ~
[DEBUG] 10:05:45.655 path.go:setMappedLocations:571 ΓåÆ mapped_locations: map[]
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:PathSeparator() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:PathSeparator() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 path.go:makeFolderFormatMap:839 ΓåÆ NO DATA
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.655 path.go:setStyle:218 ΓåÆ full
[DEBUG] 10:05:45.655 path.go:colorizePath:740 ΓåÆ cycle: []
[DEBUG] 10:05:45.655 path.go:getFolderSeparator:273 ΓåÆ NO DATA
[DEBUG] 10:05:45.655 path.go:getFolderSeparator:275 ΓåÆ \
[DEBUG] 10:05:45.655 path.go:colorizePath:743 ΓåÆ cycle_folder_separator: false
[DEBUG] 10:05:45.655 path.go:colorizePath:744 ΓåÆ %s
[DEBUG] 10:05:45.655 path.go:colorizePath:746 ΓåÆ %s
[DEBUG] 10:05:45.655 path.go:colorizePath:747 ΓåÆ %s
[DEBUG] 10:05:45.655 path.go:colorizePath:748 ΓåÆ %s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:Pwd() - 0s
[TRACE] 10:05:45.655 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.655 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.655 terminal.go:StackCount() - 0s
[DEBUG] 10:05:45.657 win32_windows.go:isWriteable:299 ΓåÆ not current user or in group
[DEBUG] 10:05:45.657 win32_windows.go:isWriteable:303 ΓåÆ current user is member of S-1-5-32-544
[DEBUG] 10:05:45.657 win32_windows.go:isWriteable:311 Γåô
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 10:05:45.657 win32_windows.go:isWriteable:313 ΓåÆ user has write access
[TRACE] 10:05:45.657 terminal_windows.go:DirIsWritable() - 1.1231ms
[TRACE] 10:05:45.657 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.666 terminal.go:CommandPath:389 ΓåÆ C:\Program Files\Git\cmd\git.exe
[TRACE] 10:05:45.666 terminal.go:CommandPath(git.exe) - 10.4238ms
[TRACE] 10:05:45.666 terminal.go:HasCommand(git.exe) - 10.4238ms
[DEBUG] 10:05:45.666 git.go:shouldDisplay:312 ΓåÆ fetch_bare_info: false
[TRACE] 10:05:45.666 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.666 terminal.go:ResolveSymlink:286 ΓåÆ C:\Users\username\Source\comunidadpy_curiosidades
[TRACE] 10:05:45.666 terminal.go:ResolveSymlink(C:\Users\username\Source\comunidadpy_curiosidades) - 549.4┬╡s
[ERROR] 10:05:45.667 terminal.go:HasParentFilePath:553 ΓåÆ CreateFile .git: The system cannot find the file specified.
[TRACE] 10:05:45.667 terminal.go:HasParentFilePath(.git) - 1.1007ms
[DEBUG] 10:05:45.667 terminal.go:CommandPath:389 ΓåÆ C:\Util\fossil.exe
[TRACE] 10:05:45.667 terminal.go:CommandPath(fossil.exe) - 11.5245ms
[TRACE] 10:05:45.667 terminal.go:HasCommand(fossil.exe) - 11.5245ms
[DEBUG] 10:05:45.719 terminal.go:RunCommand:367 Γåô
    repository:   C:\Users\username\Repositories\comunidadpy.fossil
    local-root:   C:/Users/username/Source/comunidadpy_curiosidades/
    config-db:    C:/Users/username/AppData/Local/_fossil
    checkout:     b2623c3fb893fa92ea1ce52b40d3599323b5107a 2024-09-25 16:41:03 UTC
    parent:       6c5501037c4e92f63c4f60c9fe478d8bad46f88e 2024-09-25 13:44:34 UTC
    merged-into:  7c152d467e3e6065b1cfbab94dd4aa22c4bad5a3 2024-09-25 16:41:31 UTC
    tags:         statcollector
    comment:      improve documentation (user: marcelohuerta)
    ADDED_BY_MERGE 2024-09-27.rst
    ADDED_BY_MERGE code/cli/.python-version
    ADDED_BY_MERGE code/cli/_cog_cli_init.inc
    ADDED_BY_MERGE code/cli/cyclopts.rst
    ADDED_BY_MERGE code/cli/cyclopts_cli1.py
    ADDED_BY_MERGE code/cli/cyclopts_cli2.py
    ADDED_BY_MERGE code/cli/cyclopts_cmds1.py
    ADDED_BY_MERGE code/cli/cyclopts_enum.py
    ADDED_BY_MERGE code/cli/cyclopts_except.py
    ADDED_BY_MERGE code/cli/cyclopts_except_console.html
    ADDED_BY_MERGE code/cli/cyclopts_literal.py
    ADDED_BY_MERGE code/cli/feud.rst
    ADDED_BY_MERGE code/cli/feud_cli1.py
    ADDED_BY_MERGE code/cli/feud_cli2.py
    ADDED_BY_MERGE code/cli/feud_cli3.py
    ADDED_BY_MERGE code/cli/feud_cli4.py
    ADDED_BY_MERGE code/cli/feud_cli5.py
    ADDED_BY_MERGE code/cli/feud_cmds1.py
    ADDED_BY_MERGE code/cli/feud_cmds2.py
    ADDED_BY_MERGE code/cli/genout.btm
    UPDATED_BY_MERGE code/cli/justfile
    ADDED_BY_MERGE code/cli/pyproject.toml
    DELETED    code/cli/requirements.in
    DELETED    code/cli/requirements.txt
    ADDED_BY_MERGE code/cli/savecon.btm
    ADDED_BY_MERGE code/cli/uv.lock
    ADDED_BY_MERGE code/ejemplos/test_nanodjango/pyproject.toml
    ADDED_BY_MERGE code/ejemplos/test_nanodjango/test_nanodjango.py
    ADDED_BY_MERGE code/ejemplos/test_nanodjango/uv.lock
    UPDATED_BY_MERGE code/modules/cog_helpers.py
    ADDED_BY_MERGE code/modules/setup_log.py
    UPDATED_BY_MERGE code/redate-deck.py
    UPDATED_BY_MERGE code/test_section.py
    UPDATED_BY_MERGE curiosidades.rst
    UPDATED_BY_MERGE justfile
    MERGED_WITH 8fd54527e0c0edebc2185103e53c8033d4c655037861cda6750dfff39c4310e4
[TRACE] 10:05:45.719 terminal.go:RunCommand(fossil.exe status) - 52.3135ms
[TRACE] 10:05:45.719 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.719 text.go:Render:80 ΓåÆ rendering template:  {{ .CurrentDate | date "02/01/2006 15:04:05" }}
[TRACE] 10:05:45.719 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.719 text.go:Render:80 ΓåÆ rendering template:  {{ if .WSL }}WSL at {{ end }}{{.Icon}} 
[TRACE] 10:05:45.719 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.719 text.go:Render:80 ΓåÆ rendering template: {{ if .SSHSession }}ε«⌐ {{ end }}{{ .UserName }} 
[TRACE] 10:05:45.719 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.719 text.go:Render:80 ΓåÆ rendering template:  ε¬â {{ path .Path .Location }} 
[TRACE] 10:05:45.719 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.719 text.go:Render:80 ΓåÆ rendering template:  ε£Ñ {{.Branch}} {{.Status.String}} 
[TRACE] 10:05:45.719 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.719 text.go:Render:80 ΓåÆ rendering template:  <b>{{ .Name }}</b> 
[TRACE] 10:05:45.719 terminal.go:Flags() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 517┬╡s
[DEBUG] 10:05:45.720 text.go:Render:80 ΓåÆ rendering template: powerline
[TRACE] 10:05:45.720 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.720 text.go:Render:80 ΓåÆ rendering template: powerline
[TRACE] 10:05:45.720 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.720 text.go:Render:80 ΓåÆ rendering template: {{if .Root}}green{{end}}
[TRACE] 10:05:45.720 terminal.go:Flags() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.720 text.go:Render:80 ΓåÆ rendering template: diamond
[TRACE] 10:05:45.720 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.720 text.go:Render:80 ΓåÆ rendering template: {{if .Root}}green{{end}}
[TRACE] 10:05:45.720 terminal.go:Flags() - 0s
[TRACE] 10:05:45.720 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.720 text.go:Render:80 ΓåÆ rendering template: diamond
[TRACE] 10:05:45.720 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.720 text.go:Render:80 ΓåÆ rendering template: diamond
[TRACE] 10:05:45.720 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: {{ if .Status.Changed }}p:yellow{{ end }}
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[TRACE] 10:05:45.721 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: {{ if .Status.Changed }}cyan{{ end }}
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[TRACE] 10:05:45.721 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: powerline
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: {{ if .Status.Changed }}p:yellow{{ end }}
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[TRACE] 10:05:45.721 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: {{ if .Status.Changed }}cyan{{ end }}
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[TRACE] 10:05:45.721 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: powerline
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: {{ if .Status.Changed }}p:yellow{{ end }}
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[TRACE] 10:05:45.721 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.721 text.go:Render:80 ΓåÆ rendering template: diamond
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[TRACE] 10:05:45.721 terminal.go:Shell() - 0s
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.721 python.go:Init:61 ΓåÆ environment
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.721 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.721 segment.go:SetEnabled:115 ΓåÆ segment: Python
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.721 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.721 segment.go:SetEnabled:115 ΓåÆ segment: Docker
[DEBUG] 10:05:45.721 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.721 terminal.go:Getenv(DOCKER_MACHINE_NAME) - 0s
[DEBUG] 10:05:45.721 language.go:Enabled:109 ΓåÆ extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 10:05:45.721 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[TRACE] 10:05:45.721 terminal.go:Getenv(DOCKER_HOST) - 0s
[DEBUG] 10:05:45.721 language.go:Enabled:110 ΓåÆ folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 10:05:45.721 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.721 terminal.go:Getenv(DOCKER_CONTEXT) - 0s
[DEBUG] 10:05:45.721 language.go:Enabled:117 ΓåÆ home_enabled: true
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.721 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[DEBUG] 10:05:45.721 terminal.go:Getenv:164 ΓåÆ NO DATA
[DEBUG] 10:05:45.721 segment.go:SetEnabled:115 ΓåÆ segment: Go
[TRACE] 10:05:45.721 terminal.go:Getenv(DOCKER_CONFIG) - 0s
[TRACE] 10:05:45.721 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.721 python.go:loadContext:70 ΓåÆ fetch_virtual_env: true
[DEBUG] 10:05:45.721 language.go:Enabled:109 ΓåÆ extensions: [*.go go.mod]
[DEBUG] 10:05:45.721 terminal.go:Getenv:164 ΓåÆ C:\Users\username\AppData\Local
[DEBUG] 10:05:45.721 language.go:Enabled:110 ΓåÆ folders: []
[TRACE] 10:05:45.721 terminal.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 10:05:45.721 language.go:Enabled:117 ΓåÆ home_enabled: false
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[DEBUG] 10:05:45.721 language.go:Enabled:129 ΓåÆ files
[TRACE] 10:05:45.721 terminal.go:GOOS() - 0s
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.721 segment.go:SetEnabled:115 ΓåÆ segment: Node
[DEBUG] 10:05:45.721 language.go:Enabled:109 ΓåÆ extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 10:05:45.721 language.go:Enabled:110 ΓåÆ folders: []
[DEBUG] 10:05:45.721 language.go:Enabled:117 ΓåÆ home_enabled: false
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.721 terminal_windows.go:58 ΓåÆ C:\Users\username
[DEBUG] 10:05:45.721 language.go:Enabled:129 ΓåÆ files
[DEBUG] 10:05:45.721 node.go:loadContext:56 ΓåÆ fetch_package_manager: false
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[ERROR] 10:05:45.721 terminal.go:FileContent:297 ΓåÆ open C:\Users\username\.docker\config.json: The system cannot find the path specified.
[TRACE] 10:05:45.721 terminal.go:FileContent(C:\Users\username\.docker\config.json) - 0s
[DEBUG] 10:05:45.721 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.721 terminal.go:HasFilesInDir(*.js) - 0s
[DEBUG] 10:05:45.721 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.721 terminal.go:HasFilesInDir(*.go) - 0s
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.721 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.721 terminal.go:HasFilesInDir(*.ts) - 0s
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.721 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.721 terminal.go:HasFilesInDir(go.mod) - 0s
[TRACE] 10:05:45.721 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.722 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.722 terminal.go:HasFilesInDir(package.json) - 550.2┬╡s
[TRACE] 10:05:45.722 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.722 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.722 terminal.go:HasFilesInDir(.nvmrc) - 0s
[TRACE] 10:05:45.722 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.722 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.722 terminal.go:HasFilesInDir(pnpm-workspace.yaml) - 0s
[TRACE] 10:05:45.722 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.722 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.722 terminal.go:HasFilesInDir(.pnpmfile.cjs) - 0s
[TRACE] 10:05:45.722 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.722 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.722 terminal.go:HasFilesInDir(.vue) - 0s
[DEBUG] 10:05:45.723 terminal.go:CommandPath:389 ΓåÆ C:\Program Files\Python312\python.exe
[TRACE] 10:05:45.723 terminal.go:CommandPath(python) - 1.0705ms
[DEBUG] 10:05:45.723 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.723 terminal.go:HasFilesInDir(pyvenv.cfg) - 518.2┬╡s
[DEBUG] 10:05:45.724 terminal.go:HasFilesInDir:242 ΓåÆ false
[TRACE] 10:05:45.724 terminal.go:HasFilesInDir(pyvenv.cfg) - 523.5┬╡s
[DEBUG] 10:05:45.724 python.go:loadContext:83 ΓåÆ folder_name_fallback: true
[DEBUG] 10:05:45.724 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.724 terminal.go:Getenv(VIRTUAL_ENV) - 0s
[DEBUG] 10:05:45.724 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.724 terminal.go:Getenv(CONDA_ENV_PATH) - 0s
[DEBUG] 10:05:45.724 terminal.go:Getenv:164 ΓåÆ NO DATA
[TRACE] 10:05:45.724 terminal.go:Getenv(CONDA_DEFAULT_ENV) - 0s
[TRACE] 10:05:45.724 terminal.go:Shell() - 0s
[TRACE] 10:05:45.724 terminal.go:Flags() - 0s
[TRACE] 10:05:45.724 terminal.go:Pwd() - 0s
[DEBUG] 10:05:45.724 terminal_windows.go:58 ΓåÆ C:\Users\username
[TRACE] 10:05:45.724 terminal.go:GOOS() - 0s
[DEBUG] 10:05:45.724 segment.go:SetEnabled:115 ΓåÆ segment: Text
[TRACE] 10:05:45.724 terminal.go:TemplateCache() - 0s
[DEBUG] 10:05:45.724 text.go:Render:80 ΓåÆ rendering template: Γ¥»
[TRACE] 10:05:45.724 terminal.go:Flags() - 0s
[DEBUG] 10:05:45.724 text.go:Render:80 ΓåÆ rendering template: plain
[TRACE] 10:05:45.724 terminal.go:Flags() - 0s
[TRACE] 10:05:45.724 terminal.go:Shell() - 0s
[TRACE] 10:05:45.724 terminal.go:Flags() - 0s
[TRACE] 10:05:45.724 terminal.go:Shell() - 0s
[DEBUG] 10:05:45.724 terminal.go:Getenv:164 ΓåÆ C:\Users\username\AppData\Local
[TRACE] 10:05:45.724 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 10:05:45.724 terminal.go:CachePath() - 0s
[TRACE] 10:05:45.724 terminal.go:Flags() - 0s
```

--- a/src/segments/fossil.go
+++ b/src/segments/fossil.go
@@ -11,9 +11,9 @@ func (s *FossilStatus) add(code string) {
 	switch code {
 	case "CONFLICT":
 		s.Conflicted++
-	case "DELETED":
+	case "DELETED", "MISSING":
 		s.Deleted++
-	case "ADDED", "ADDED_BY_INTEGRATE":
+	case "ADDED", "ADDED_BY_INTEGRATE", "ADDED_BY_MERGE":
 		s.Added++
 	case "EDITED", "UPDATED", "UPDATED_BY_INTEGRATE", "UPDATED_BY_MERGE", "CHANGED":
 		s.Modified++

--- a/src/segments/fossil_test.go
+++ b/src/segments/fossil_test.go
@@ -45,8 +45,10 @@ func TestFossilStatus(t *testing.T) {
 			comment:      In the /setup_skin page, add a mention of/link to /skins, per request in the forum. (user: stephan)
 			CONFLICT             test.tst
 			DELETED	             test.tst
+			MISSING              test.tst
 			ADDED                test.tst
 			ADDED_BY_INTEGRATE   test.tst
+			ADDED_BY_MERGE       test.tst
 			EDITED               auto.def
 			UPDATED              test.tst
 			UPDATED_BY_INTEGRATE test.tst
@@ -55,7 +57,7 @@ func TestFossilStatus(t *testing.T) {
 			RENAMED 		     test.tst
 			`,
 			ExpectedBranch: "trunk",
-			ExpectedStatus: "+2 ~5 -1 >1 !1",
+			ExpectedStatus: "+3 ~5 -2 >1 !1",
 		},
 	}
 	for _, tc := range cases {

